src/header.iced

```diff
@@ -75,23 +84,25 @@ exports.parse_encryption_header_packet = (decryptor, header_intermediate) ->
   found = false
   payload_key = null
   for recipient_index in [0...recipients.length]
-    if util.bufeq_secure(recipients[recipient_index][0], decryptor.publicKey)
+    if recipients[recipient_index][0] is null then continue
+    else if util.bufeq_secure(recipients[recipient_index][0], decryptor.publicKey)
       payload_key = decryptor.box_open_afternm({ciphertext : recipients[recipient_index][1], nonce : nonce.nonceForPayloadKeyBox(), secret})
       found = true
       break
 
   #check for anonymous recipients
   if not found then for recipient_index in [0...recipients.length]
     try
-      payload_key = decryptor.box_open_afternm({ciphertext : recipients[recipient_index], nonce : nonce.nonceForPayloadKeyBox(), secret})
+      payload_key = decryptor.box_open_afternm({ciphertext : recipients[recipient_index][1], nonce : nonce.nonceForPayloadKeyBox(), secret})
+      break if payload_key?
```

Comment:
I don't *think* so - my tests never hit it - but I have seen sodium/tweetnacl return null/empty buffer without erroring, so I wanted to cover that possibility